apps/prairielearn/src/lib/workspace.js

```diff
@@ -451,6 +486,19 @@ module.exports = {
       }
     }
 
+    issues.forEach(async (issue) => {
+      await sqldb.callAsync('issues_insert_for_variant', [
+        variant.id,
+        'Error initializing workspace', // Student message
+        issue.msg, // Instructor message
+        false, // Manually reported
+        true, // Course caused
+        { workspace, variant, question, course, error: issue.data }, // Course data
+        issue.data, // System data
+        null, // Authenticated user
+      ]);
+    });
```

Comment:
This will not properly handle errors. I'd use [`async.eachSeries`](https://caolan.github.io/async/v3/docs.html#eachSeries) instead.

Reply:
Though given my suggestion to consolidate all the issue reports into a single issue, this comment would no longer be relevant.